src/icons/AlertTriangleFilled.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg fill="none" version="1.1" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="m12 2.8965c-0.6991 0-1.3483 0.36517-1.7109 0.96289l-8.4687 14.141c-0.35536 0.6154-0.35777 1.3729-0.0058594 1.9902 0.3519 0.6174 1.0043 1.002 1.7148 1.0098h16.941c0.7106-0.0078 1.3629-0.39237 1.7148-1.0098 0.3519-0.6173 0.34954-1.3748-0.005859-1.9902l-8.4688-14.141c-0.3625-0.59772-1.0118-0.96289-1.7109-0.96289zm0 5.6035a0.5 0.5 0 0 1 0.5 0.5v4a0.5 0.5 0 0 1-0.5 0.5 0.5 0.5 0 0 1-0.5-0.5v-4a0.5 0.5 0 0 1 0.5-0.5zm0 7.5a1 1 0 0 1 1 1 1 1 0 0 1-1 1 1 1 0 0 1-1-1 1 1 0 0 1 1-1z" fill="#000"/>
+</svg>
```

Comment:
This seems to be different from the icon in Compound?

![Screenshot 2022-06-09 at 17-58-24 Figma](https://user-images.githubusercontent.com/48614497/172952366-3e93a7b3-a3fa-4967-9576-afc0a64ab819.png)

Reply:
Ah, for some reason I only looked at the icons we were using in element-web rather than looking in compound - replaced.